autobotAI_integrations/integrations/__init__.py

```
@@ -29,34 +137,85 @@ def get_services(self):
     def get_ai_services(self):
         return list(self._ai_services.keys())
 
-    def get_service_details(self, q=None):
+    def get_service_details(self, q=None, include_details=True):
```

Ruff (0.15.13)

[warning] Boolean default positional argument in function definition

(FBT002)